core/templates/list.h

```diff
@@ -761,6 +763,12 @@ class List {
 		}
 	}
 
+	List(std::initializer_list<T> init) {
+		for (const auto& i : init) {
```

Comment:
```suggestion
		for (const T &i : init) {
```
`auto` is [not allowed](https://docs.godotengine.org/en/stable/contributing/development/cpp_usage_guidelines.html#auto-keyword)

Reply:
Fixed.